pages/index.js

```diff
@@ -31,6 +31,12 @@ const featuredLinksArray = [
     imageSource: `${s3}redesign/images/meetup-lecture.jpg`,
     alt: 'Dozens of developers look at a lecturer.',
   },
+  {
+    href: 'podcasts',
+    name: 'Podcasts',
+    imageSource: `${s3}redesign/images/fist-bumping.jpg`, // change photo
+    lt: 'A team fist-bumping eachother over a table.',
```

Comment:
missed an `a` in `alt`

Reply:
Fixed it locally, but before I commit - is there a different stock image we could use? I see that a lot of our stock images are already being used.

Reply:
We can leave it as is for now, and I'll find a better one last minute.